plugins/hls-class-plugin/src/Ide/Plugin/Class/CodeAction.hs

```diff
@@ -198,8 +198,9 @@ codeAction recorder state plId (CodeActionParams _ _ docId _ context) = do
                         _ -> fail "Ide.Plugin.Class.findClassFromIdentifier"
         findClassFromIdentifier _ (Left _) = throwError (PluginInternalError "Ide.Plugin.Class.findClassIdentifier")
 
-isClassNodeIdentifier :: IdentifierDetails a -> Bool
-isClassNodeIdentifier ident = (isNothing . identType) ident && Use `Set.member` identInfo ident
+isClassNodeIdentifier :: Identifier -> IdentifierDetails a -> Bool
```

Comment:
Were you able to reproduce the issue? It would be great (and shouldn't be that hard) to add a test that reproduces the original issue and confirms this fix works.